atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SerializableTransaction.java

```diff
@@ -638,9 +639,20 @@ private void verifyColumnRanges(Transaction readOnlyTransaction) {
                     NavigableMap<Cell, ByteBuffer> readsInRange = Maps.transformValues(
                             getReadsInColumnRange(table, row, range),
                             input -> ByteBuffer.wrap(input));
-                    boolean isEqual = bv.transformBatch(input -> filterWritesFromCells(input, writes))
-                            .isEqual(readsInRange.entrySet());
+                    List<Entry<Cell, ByteBuffer>> realValuesAsList =
+                            bv.transformBatch(input -> filterWritesFromCells(input, writes)).immutableCopy();
+                    List<Entry<Cell, ByteBuffer>> readValuesAsList = ImmutableList.copyOf(readsInRange.entrySet());
+                    boolean isEqual = realValuesAsList.equals(readValuesAsList);
                     if (!isEqual) {
+                        log.info("Failing a serializable transaction because column ranges not equal",
+                                UnsafeArg.of("table", table),
```

Comment:
You can use `LoggingArgs...` here to make this safe if you want, not essential though

Reply:
oh interesting, I knew there was a class like that. Meh at this point I want it merged fast enough that a rebuild is annoying.